packages/peregrine/lib/talons/Footer/useFooter.js

```diff
@@ -1,12 +1,15 @@
 import { useQuery } from '@apollo/client';
 
+import DEFAULT_OPERATIONS from './footer.gql';
+
 /**
  *
- * @param {*} props.query the footer data query
+ * @param {*} props.operations GraphQL operations used by talons
  */
-export const useFooter = props => {
-    const { query } = props;
-    const { data } = useQuery(query);
+export const useFooter = (props = {}) => {
```

Comment:
Discussion point: what is the preferred way to handle talons where `operations` is the **only** prop? We can either:

1. Do it this way and default to an empty object so operation destructuring below works
2. Don't defaults props to empty object and force that usage of talon with no operation override do `const talonProps = useFooter({})`

I have a preference for option 1.

Reply:
Option 1 :D